soak/helm/ack-soak-test/templates/job.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: batch/v1
+kind: Job
```

Comment:
I thought that the goal was keep these tests running for ever/periodically. Maybe we could use a `batch/v1.CronJob` instead?

Reply:
No, that's not the idea.
These tests will run once for a longer duration.